src/util-streaming-buffer.c

```diff
@@ -2366,6 +2374,44 @@ static int StreamingBufferTest11(void)
     StreamingBufferFree(sb, &cfg);
     PASS;
 }
+
+static const char *dummy_conf_string = "%YAML 1.1\n"
+                                       "---\n"
+                                       "\n"
+                                       "app-layer:\n"
+                                       "  protocols:\n"
+                                       "    http:\n"
+                                       "      enabled: yes\n"
+                                       "      memcap: 88\n"
```

Comment:
My staging logic, which also tests on 32bit arm, fails here. I assume because of this memcap value being tuned for 64 bit.

Reply:
@victorjulien thanks, have you got any suggestions of how it can be fixed? I don't have a 32bit arm system setup. So unless I do all that, I'm not sure the best solution at this moment.

Reply:
I've been playing with this a bit, and it seems the most reliable way to do it is just to increase the seg2 data. I made it 52 bytes to have some margin, and it now passes on arm7

Reply:
Great, thanks. I will update the code and create a new PR. I presume for tidiness sake you just created an array[57]?

Reply:
I just tested with
![image](https://github.com/OISF/suricata/assets/32410611/eac0e8a2-6f9e-4d17-a0ed-245d4ccd127c)